apps/backend/src/main.ts

```diff
@@ -36,6 +39,15 @@ async function bootstrap() {
     })
   );
   app.use(json({limit: '50mb'}));
+  app.use(
+    session({
+      secret: configService.get('JWT_SECRET') || generateDefault(),
+      saveUninitialized: true,
+      resave: true
```

Comment:
Similar to above, based on the docs it may be useful to set this to false if possible since this causes the session to be saved over and over, which we may not need since we are just using this for login sessions.